testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/beanvalidation/hibernate/validator/BootStrapValidationTestCase.java

```diff
@@ -56,6 +57,8 @@
     public static Archive<?> deploy() {
         final WebArchive war = ShrinkWrap.create(WebArchive.class, "testbootstrapvalidation.war");
         war.addPackage(BootStrapValidationTestCase.class.getPackage());
+        // Hibernate Validator needs the following runtime permission
+        war.addAsManifestResource(createPermissionsXmlAsset(new RuntimePermission("accessDeclaredMembers")), "permissions.xml");
```

Comment:
After discussion with Hibernate team, I've filed a Hibernate Jira for the Validator issue, see https://hibernate.atlassian.net/browse/HHH-10349

Thus, this PR is no longer valid. I will create a new one.